devel/git-latexdiff/Portfile

```diff
@@ -18,11 +18,13 @@ long_description    Tool to graphically visualize differences between \
                     and latexdiff.
 
 homepage            https://gitlab.com/${name}/${name}
-master_sites        https://gitlab.com/${name}/${name}/repository/archive.tar.gz?ref=v${version}&dummy=
-distname            ${name}-v${version}-2bd0dcbf2a14de7c311382ed03eb24bac661ac69
+master_sites        https://gitlab.com/${name}/${name}/-/archive/v${version}
+distname            ${name}-v${version}
 
-checksums           rmd160  29b61b526bbed34d32ed8d945c3e569a36d10fc7 \
-                    sha256  76b19b623749025bde0617776237dbd6763a21d19cd6b6b353765846dac766ec
+
```

Comment:
The file didn't need this extra newline, but I'm not going to hold up committing it for this. If you update again in the future, get rid of it. 😀